soh/soh/Enhancements/enhancementTypes.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
🔥 been meaning to address this when someone found it the other day.
Although I would prefer the `#ifndef FILE_H` pattern over `#pragma once`